src/components/SingleEvent/Admin/PartyForm.js

```diff
@@ -9,30 +64,53 @@ class PartyForm extends Component {
       name = '',
       description = '',
       location = '',
-      date = '',
+      date = `${new Date().getTime()}`,
       image = '',
       deposit = '0.02',
       coolingPeriod = `${60 * 60 * 24 * 7}`,
       limitOfParticipants = 20
     } = props
+
+    const [day, time] = getDayAndTimeFromDate(date)
+
     this.state = {
       name,
       description,
       location,
-      date,
+      day: new Date(day),
+      time: moment(time),
       image,
       deposit,
       coolingPeriod,
-      limitOfParticipants
+      limitOfParticipants,
+      imageUploading: false
     }
   }
 
+  onDrop = acceptedFiles => {
+    acceptedFiles.forEach(file => {
+      const reader = new FileReader(file)
+      reader.onload = () => {
+        const dataUrl = reader.result
+        this.setState({ imageUploading: true })
+        upload(dataUrl).then(url => {
+          this.setState({ image: url })
+        })
+      }
+      reader.onabort = () => console.log('file reading was aborted')
+      reader.onerror = () => console.log('file reading has failed')
```

Comment:
Are they more of `console.warn`?

Reply:
I wonder if it's worth doing logging via a centralized logging manager, like we do for the server. Would make it easy to capture these logs for cloud storage later on.

Reply:
I can change them to console.warn. I think if we're going to do a centralised logging manager it should be a separate PR. If we did - is there any type of solution you would suggest?